database.hh

```diff
@@ -312,9 +312,35 @@ struct cf_stats {
 
 class table;
 using column_family = table;
+struct table_stats;
+using column_family_stats = table_stats;
 
 class database_sstable_write_monitor;
 
+struct table_stats {
+    /** Number of times flush has resulted in the memtable being switched out. */
+    int64_t memtable_switch_count = 0;
+    /** Estimated number of tasks pending for this column family */
+    int64_t pending_flushes = 0;
+    int64_t live_disk_space_used = 0;
+    int64_t total_disk_space_used = 0;
+    int64_t live_sstable_count = 0;
+    /** Estimated number of compactions pending for this column family */
+    int64_t pending_compactions = 0;
+    int64_t partition_creations = 0;
```

Comment:
Looks like you snuck in some extra changes thinking no one would notice.

Reply:
Sorry, I messed up a little during the rebase. New counters are properly introduced in the second commit, and the issue with returning void is resolved, too.